app/javascript/packages/form-steps/form-steps.tsx

```diff
@@ -169,6 +170,17 @@ function useStepTitle(step: FormStep, titleFormat?: string) {
   }, [step]);
 }
 
+function trackStepEvents(step: FormStep) {
```

Comment:
By convention, I would expect a custom hook to begin with "use".

>A custom Hook is a JavaScript function whose name starts with ”use” and that may call other Hooks.

https://reactjs.org/docs/hooks-custom.html#extracting-a-custom-hook